### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by kumberto**

### CODE REVIEW

File: EventsAPI/EventsAPI.cpp [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+	SetEvent(h);
+	ResetEvent(h2);
+
+	return 0;
+}
+int main()
+{
+	h = CreateEventA(0, FALSE, FALSE, "MyClient");
+	h2 = CreateEventA(0, FALSE, FALSE, "MyServer");
```
Comment (ak-toktor) [inline EventsAPI/EventsAPI.cpp:38]: you should check the results

File: EventsAPI/EventsAPI.cpp [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#include <windows.h>
+
+HANDLE h = INVALID_HANDLE_VALUE;
+HANDLE h2 = INVALID_HANDLE_VALUE;
+std::vector<int> data;
+
+DWORD WINAPI ClientTread(LPVOID arg)
+{
+	SetEvent(h2);
+	DWORD res = WaitForSingleObject(h, INFINITE);
```
Comment (ak-toktor) [inline EventsAPI/EventsAPI.cpp:15]: you should check the result.

File: EventsAPI/EventsAPI.cpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+}
+int main()
+{
+	h = CreateEventA(0, FALSE, FALSE, "MyClient");
+	h2 = CreateEventA(0, FALSE, FALSE, "MyServer");
+
+	CreateThread(0, 0, ClientTread, 0, 0, 0);
+	CreateThread(0, 0, ServerTread, 0, 0, 0);
+
+	Sleep(10000);
```
Comment (ak-toktor) [inline EventsAPI/EventsAPI.cpp:43]: It is better to use WaitForMultipleObjects

File: EventsAPI/EventsAPI.cpp (hunk 2 above)
Comment (kumberto) [reply EventsAPI/EventsAPI.cpp:15]: ok

File: EventsAPI/EventsAPI.cpp (hunk 1 above)
Comment (kumberto) [reply EventsAPI/EventsAPI.cpp:38]: ok

File: EventsAPI/EventsAPI.cpp (hunk 3 above)
Comment (kumberto) [reply EventsAPI/EventsAPI.cpp:43]: ok

**Pull request merged**